qi-lib/pass/deforest.rkt

```diff
@@ -147,22 +147,22 @@
                                 (#%host-expression post-arg) ...))
                               blanket?))
                    form-stx)
-      #:attr next #'range->cstream-next
-      #:attr prepare #'range->cstream-prepare
-      #:attr contract #'(->* (real?) (real? real?) any)
-      #:attr name #'range
-      #:attr curry (make-producer-curry 1 3
-                                        blanket? pre-arg post-arg
-                                        fine? arg
-                                        form-stx))
+             #:attr next #'range->cstream-next
+             #:attr prepare #'range->cstream-prepare
+             #:attr contract #'(->* (real?) (real? real?) any)
+             #:attr name #'range
+             #:attr curry (make-producer-curry 1 3
+                                               blanket? pre-arg post-arg
+                                               fine? arg
+                                               form-stx))
```

Comment:
I think these are pure indentation changes? Unless you have a strong preference for the new indentation, would prefer to leave these changes out. We can standardize on project local indentation at some point, as we talked about recently (I've also mentioned it [in the wiki](https://github.com/drym-org/qi/wiki/Coding-Conventions)) and looks like there is a config for `pattern` there that you could use in Emacs). We should probably try to standardize on whatever DrRacket does, unless it does something we don't like 😄